### ISSUE TITLE
mongoose 6.8 compatibility

### ISSUE THREAD

**Issue by cylwin** [typegoose/auto-increment#17]
Hello, 

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: xxx@1.0.0
npm ERR! Found: mongoose@6.8.0
npm ERR! node_modules/mongoose
npm ERR!   mongoose@"^6.8.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mongoose@"~6.6.0" from @typegoose/auto-increment@1.8.0
npm ERR! node_modules/@typegoose/auto-increment
npm ERR!   @typegoose/auto-increment@"*" from the root project```

**Comment by hasezoey** [comment]
thanks for noticing that this package was not updated yet for these versions

i will release a version for mongoose 6.7 soon, but as for 6.8 i will wait until typegoose 10 is released (which is slated for 12.12.22 d/m/y)

**Comment by hasezoey** [comment]
for now version [1.9.0 has been released](https://github.com/typegoose/auto-increment/releases/tag/v1.9.0) with mongoose 6.7 support
